test/policy.json

```diff
@@ -3,5 +3,21 @@
         {
             "type": "insecureAcceptAnything"
         }
-    ]
+    ],
+    "transports": {
+        "docker": {
+            "docker.io/library/hello-world": [
+                {
+                    "type": "reject"
+		}
+	    ],
```

Comment:
These two lines use hard tabs instead of spaces, making the alignment inconsistent.